src/app/content/components/Content.tsx

```diff
@@ -165,51 +144,63 @@ const OuterWrapper = styled.div`
  *   of things need to know when the sidebar is open/closed.
  */
 // tslint:disable-next-line:variable-name
-const Content = ({mobileExpanded}: {mobileExpanded: boolean}) => <Layout>
-  <ScrollOffset
-    desktopOffset={
-      bookBannerDesktopMiniHeight
-      + toolbarDesktopHeight
-      + scrollOffset
-    }
-    mobileOffset={
-      bookBannerMobileMiniHeight
-      + (mobileExpanded ? toolbarMobileExpandedHeight : toolbarMobileHeight)
-      + scrollOffset
+const Content = ({mobileExpanded, openToc}: {mobileExpanded: boolean, openToc: () => void}) => {
+  const isMobile = useMatchMobileMediumQuery();
+
+  React.useEffect(() => {
+    if (!isMobile) {
+      openToc();
     }
-  />
-  <Background>
-    <BookBanner />
-    <ErrorBoundary>
-      <HighlightsPopUp />
-      <StudyguidesPopUp />
-      <PracticeQuestionsPopup />
-      <NudgeStudyTools />
-      <Toolbar />
-      <OuterWrapper>
-        <SearchResultsSidebar/>
-        <Wrapper>
-          <CenteredContentRow>
-            <TableOfContents />
-            <ContentPane>
-              <UndoPadding>
-                <MainContentWrapper>
-                  <ContentNotifications mobileExpanded={mobileExpanded} />
-                  <Page />
-                  <Attribution />
-                  <Footer/>
-                </MainContentWrapper>
-              </UndoPadding>
-            </ContentPane>
-          </CenteredContentRow>
-        </Wrapper>
-      </OuterWrapper>
-    </ErrorBoundary>
-  </Background>
-</Layout>;
+  }, []);
```

Comment:
@TomWoodward this is so `tocOpen` doesn't stay set to null when Content renders on desktop

Reply:
ok, but this produces a mis-alignment. if we're going to give up on css and set the state with js, then we should do that for the whole sidebar. the way this is set up, if i load the page on mobile, then enlarge the window, the toc does appear but the button state becomes wrong.

i prefer the css approach to the default state, but i'm not sure what the issue you're having is with the button

Reply:
I removed the state logic and fixed the issue in css